style.css

```diff
@@ -0,0 +1,81 @@
+@import url('https://fonts.googleapis.com/css2?family=Playfair+Display&family=Poppins&display=swap');
+
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+}
+
+body {
+  position: relative;
+  width: 320px;
+  height: 4910px;
+  background: #fff;
+}
+
+.geomtry2 {
+  position: absolute;
+  width: 320px;
+  height: 160px;
+  left: 0;
+  top: 472px;
+}
+
+.menu {
+  position: absolute;
+  width: 20px;
+  height: 14px;
+  left: 284px;
+  top: 19px;
+}
+
+.title {
+  position: absolute;
+  width: 220px;
+  height: 100px;
+  left: 16px;
+  top: 104px;
+  font-family: 'Playfair Display', serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 50px;
+  line-height: 100%;
+  color: #1a2236;
+}
+
+.paragraph {
+  position: absolute;
+  width: 288px;
+  height: 126px;
+  left: 16px;
+  top: 224px;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 14px;
+  line-height: 21px;
+  letter-spacing: -0.01em;
+  color: #1a2236;
+}
+
+.bttn {
+  left: 16px;
+  top: 390px;
+  width: 288px;
+  height: 50px;
+  font-size: 16px;
+  font-weight: 500;
+  color: #fff;
+  line-height: 24px;
+  text-align: center;
+  font-style: normal;
+  position: absolute;
+  box-sizing: border-box;
+  font-family: 'Poppins', serif;
+  background-color: #1a2236;
+}
```

Comment:
- [ ] Kindly follow the button styling: Active, Hover and Pressed (see design).
![hoverfigma](https://user-images.githubusercontent.com/66992901/198533830-48920119-d44d-4cf9-aa7e-dbf3b5cb647c.png)